egs/callhome_diarization/v1/diarization/kaldi_io.py

```diff
@@ -0,0 +1,627 @@
+#!/usr/bin/env python
```

Comment:
I don't think diarization/ directory is the best place for this. It probably belongs somewhere else. Possibly the code should be downloaded from wherever its hosted, rather than added to our repository. @danpovey would you have a suggestion? I believe this is a general set of tools for handling Kaldi files in python. In this PR, it's used by BUT's VB resegmentation code.

Reply:
@HuangZiliAndy you should probably make a script in tools/extras, e.g., tools/extras/install_kaldi_io_py.sh that installs this. Probably you can use install_tensorflow_py.sh as an example.